packages/components-js/projects/partials/scripts/generateInitialStylesPartial.ts

```diff
@@ -13,6 +23,61 @@ type GetInitialStylesOptionsFormatHtml = Omit<GetInitialStylesOptions, 'withoutT
 type GetInitialStylesOptionsFormatJsx = Omit<GetInitialStylesOptions, 'withoutTags'> & { format: 'jsx' };
 type GetInitialStylesOptionsWithoutTags = Omit<GetInitialStylesOptions, 'format'>;`;
 
+  const normalizeStyles: Styles = {
+    '@global': {
+      '*': {
+        fontFamily: fontFamily,
+        lineHeight: fontLineHeight,
+        ...fontBehavior,
+      },
+
+      p: {
+        fontWeight: fontWeight.regular,
+        ...fontHyphenation,
+      },
+
+      'h1, h2, h3, h4, h5, h6': {
+        fontWeight: fontWeight.regular,
+      },
+
+      a: {
+        color: 'inherit',
+        textDecoration: 'underline',
+        '@media(hover:hover)': {
+          transition: 'color var(--p-transition-duration, .24s) ease',
+          '&:hover': {
+            color: themeLight.state.hover,
+          },
+        },
+      },
+
+      '@media(hover:hover)': {
+        '[data-theme="dark"] a:hover': {
+          color: themeDark.state.hover,
+        },
+      },
+
+      'b, strong': {
+        fontWeight: fontWeight.bold,
+      },
+
+      'em, i': {
+        fontStyle: 'normal',
```

Comment:
Some of the styles don't match v3 specification anymore. Will this be refactored later?